dev/com.ibm.ws.org.apache.cxf.cxf.core.3.2/src/org/apache/cxf/interceptor/MessageSenderInterceptor.java

```diff
@@ -0,0 +1,87 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.cxf.interceptor;
+
+import java.io.IOException;
+import java.net.SocketTimeoutException;
+import java.util.ResourceBundle;
+
+import org.apache.cxf.common.i18n.BundleUtils;
+import org.apache.cxf.message.Exchange;
+import org.apache.cxf.message.Message;
+import org.apache.cxf.phase.AbstractPhaseInterceptor;
+import org.apache.cxf.phase.Phase;
+import org.apache.cxf.transport.Conduit;
+import org.apache.cxf.common.logging.LogUtils;
+import java.util.logging.Logger;
+
+/**
+ * Takes the Conduit from the exchange and sends the message through it.
+ */
+public class MessageSenderInterceptor extends AbstractPhaseInterceptor<Message> {
+    private static final ResourceBundle BUNDLE = BundleUtils.getBundle(MessageSenderInterceptor.class);
+    private MessageSenderEndingInterceptor ending = new MessageSenderEndingInterceptor();
+
+    private static final Logger LOG = LogUtils.getL7dLogger(MessageSenderInterceptor.class);  // Liberty Change
```

Comment:
Why is this added if there is no usage of LOG?   Are there other Liberty changes in this class?

Reply:
I meant to add trace, but didn't find any good reason to add additional trace here, so will remove this LOG declaration.